tests/arista_eos/show_interfaces_transceiver/arista_eos_show_interfaces_transceiver.raw

```diff
@@ -0,0 +1,12 @@
+If device is externally calibrated, only calibrated values are printed.
+N/A: not applicable, Tx: transmit, Rx: receive.
+mA: milliamperes, dBm: decibels (milliwatts).
+                               Bias      Optical   Optical                
+          Temp       Voltage   Current   Tx Power  Rx Power               
+Port      (Celsius)  (Volts)   (mA)      (dBm)     (dBm)     Last Update  
+-----     ---------  --------  --------  --------  --------  -------------------
```

Comment:
Let's do a state change here, and be much looser in the actual capture groups. 

something like (not tested) e.g.

```
Value PORT (\S+)
Value TEMP (\S+)
Value VOLTAGE (\S+)
Value BIAS_CURRENT (\S+)
Value TX_POWER (\S+)
Value RX_POWER (\S+)


Start
  ^---- > Begin

Begin
 ^${PORT}\s+${TEMP}\s+${VOLTAGE}\s+${BIAS_CURRENT}\s+${TX_POWER}\s+${RX_POWER}\s+(\S+ ago|N/A) -> Record

```

Reply:
@itdependsnetworks , thanks for review. I was on vacations and could not get this done earlier . Please review suggested changes and approve as appropriate